Cannot find module 'underscore'

I'm new to Keystone and just tried this generator for the first time. When I run `node keystone` I get

````
module.js:327
    throw err;

Error: Cannot find module 'underscore'
````

Steps to reproduce:

1. Run `yo keystone` with the following configuration
![screen shot 2016-07-11 at 12 06 19](https://cloud.githubusercontent.com/assets/713106/16727164/1340d51a-4760-11e6-97a4-9cdf228e5b2f.png)
2. Run `node keystone`

Searching through the generated files I can find references to underscore in `templates/views/helpers/index.js`.

Workaround:
`npm install --save underscore` will fix this problem

--- a/app/templates/_package.json
+++ b/app/templates/_package.json
@@ -16,7 +16,8 @@
     "node-sass": "^3.3.2",
     "node-sass-middleware": "^0.9.7", <% } else if (preprocessor === 'stylus') { %>
     "stylus": "^0.53.0", <% } %>
-    "dotenv": "^2.0.0"
+    "dotenv": "^2.0.0",
+    "underscore": "^1.8.3"
   },
   "devDependencies": {
     "eslint": "^2.12.0",

